### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by jeremyjonas867** [smartcontractkit/chainlink#1794]
Hello. After successfully setting up a Chainlink node, I'm failing tests with the following output

```
 FAIL  __tests__/containers/SignIn.test.js
  ● containers/SignIn › unauthenticated user can input credentials and sign in

    fetch-mock: No fallback response defined for POST to 6688/sessions

       7 | ): Promise<Response> {
       8 |   return Promise.race([
    >  9 |     fetch(url, options),
         |     ^
      10 |     (new Promise((_, reject) =>
      11 |       setTimeout(() => reject(new Error('timeout')), timeout),
      12 |     ) as any) as Response,
```

### System Information

- Go version: 1.13.1
- Host OS: Debian
- Commit (INFO line when starting the node): 6a2d5e4ce40ac22e7b72e12f2fcd6545d1a5049c

This happens on a newly cloned repository, in operator-ui while running `npm run test`

**Comment by thodges-gh** [comment]
@jeremyjonas867 I'm also on Debian and get passing tests when cloning to a new directory. What version of node and yarn are you running?

Here are mine:
```
$ node -v
v10.15.3
$ yarn -v
1.19.1
```

**Comment by HenryNguyen5** [comment]
This is related to how we're forming urls for requests, @rupurt was running into the same issue. At least for now, you can work around this test failure by making sure you have `CHAINLINK_PORT` env var undefined before running the test.

**Comment by thodges-gh** [comment]
Yep, once I set CHAINLINK_PORT, I get failing tests.

**Comment by hellobart** [comment]
Fixed by #1896

**Comment by rupurt** [comment]
Top stuff @hellobart
